### ISSUE TITLE
Full GO annotation database download

### ISSUE THREAD

**Issue by stuppie** [geneontology/helpdesk#133]
I'm looking at the download page here: http://www.geneontology.org/page/go-mysql-database-guide
For the "full GO database", it says it is updated monthly, however the files in http://archive.geneontology.org/latest-full/ were last updated 07-Jan-2017. Is there a more up to date source for the full annotation download? Thank you in advance!

**Comment by cmungall** [comment]
We'll have a more general announcement going out soon - for your use case, is this for loading wikidata?

**Comment by stuppie** [comment]
Yes, I'm just trying to find a way to get all of the annotations for a particular species (by taxid). From amigo, it looks like I can only query by organism name. I suppose I can do a SQL query though ...

**Comment by kltm** [comment]
As well, from #140 : 

The legacy systems (mainly the pipeline and delivery of the MySQL, mentioned in #4 ) are being decommissioned as they fail or no longer become viable. The new system is in various states of readiness, with the rdf endpoint (rdf.geneontology.org) being largely ready to use, with the peripheral query interface and documentation still being built up.
A couple of good places to start with the new system would be:

https://github.com/geneontology/go-site/blob/master/graphstore/triplestore_info.md
https://github.com/geneontology/go-graphstore/wiki/Example-queries

We're currently working on more documentation and introductions. Our apologies for the awkward transition period.

**Comment by stuppie** [comment]
Thank you for the info. I did see this page before but was unable to figure out how to get annotations from only a particular species. Is that info contained in this graph?
Also, none of the examples on the [example queries page](https://github.com/geneontology/go-graphstore/wiki/Example-queries) work, i.e. I get 0 results (or I'm not doing something right). I'm executing them on this page: http://rdf.geneontology.org/blazegraph/#query)

**Comment by suzialeksander** [comment]
@dougli1sqrd @ktlm any ideas on why the example queries are not working?

**Comment by kltm** [comment]
@stuppie Apologies.
The queries on that page are from an earlier version of the public graphstore we run, which likely now has slightly different identifiers and structure.
The most up-to-date information should be found here:
https://github.com/geneontology/go-site/blob/master/graphstore/triplestore_info.md
With some templated examples here:
https://github.com/geneontology/sparqlr/tree/master/templates
